Docs: Add some more details to running from source doc (#1680)
ArchipelagoMW/Archipelago: docs/running from source.md

 * Download and install a "Windows installer (64-bit)" from the [Python download page](https://www.python.org/downloads)
   * **Python 3.11 does not work currently**

>  * (**Optional**) Download and install Visual Studio Build Tools from

Reviewer: I think the **(bold)** looks odd. Just personal taste though
```suggestion
 * **Optional**: Download and install Visual Studio Build Tools from
```

   [Visual Studio Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/).
   * Refer to [Windows Compilers on the python wiki](https://wiki.python.org/moin/WindowsCompilers) for details. 
     Generally, selecting the box for "Desktop Development with C++" will provide what you need.

>    * Instead of installing the build tools if not building the program, pre-compiled modules are pinned on

Reviewer: Maybe it makes more sense like this?
```suggestion
   * Build tools are not required if all modules are installed pre-compiled. Pre-compiled modules are pinned on
```
